pkg/messaging/sns.go

```diff
@@ -6,47 +6,44 @@ import (
 	_ "github.com/aws/aws-sdk-go/aws/credentials/stscreds"
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/sns"
+	"strings"
 )
 
-// NewSNS initializes a new AWS SNS client
-func NewSNS(config *Config) (*sns.SNS, error) {
+// NewSNS initializes a new AWS SNS client and returns topicArn - default for EmailTopic topic
```

Comment:
this should not have a default... as mentioned before... if this is in packages it should have no concept of the consuming system... if they don't pass in a topic it should error... also, you're setting up variadic parameters, but only using param 0... so that should just be changed to ` topic string` and not variadic

Reply:
For to change the desc of this function - there's no default (EmailTopic) any more.
Yes you are right for variadic - will change to pass topic string